src/gui.cpp

```diff
@@ -246,6 +246,10 @@ void Gui_Render()
     qglPixelStorei(GL_UNPACK_LSB_FIRST, GL_FALSE);
     qglPixelStorei(GL_UNPACK_ALIGNMENT, 1);
 
+    if (screen_info.show_debuginfo != 0)
```

Comment:
here should be something like 'if(screen_info.show_crosshair)'; compare crosshair and debug info is a hack;